esp/src/eclwatch/templates/QuerySetQueryWidget.html

```diff
@@ -33,6 +33,22 @@
                             <option value="0">${i18n.NotActive}</option>
                         </select>
                     </div>
```

Comment:
I would add <span data-dojo-type="dijit.ToolbarSeparator"></span> here to separate the controls on the toolbar.